src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -176,12 +176,12 @@ def __str__(self):
 # Both the alloc and free function need to be whitelisted.
 
 mismatch_table = {'client': ('array'),
-                  'daos': ('common', 'container', 'pool'),
+                  'daos': ('common', 'container', 'pool', 'server'),
                   'common': ('container', 'pool'),
-                  'container': ('common'),
+                  'container': ('common', 'server'),
                   'mgmt': ('common', 'daos', 'pool', 'rsvc'),
                   'misc': ('common', 'mgmt'),
-                  'pool': ('common'),
+                  'pool': ('common', 'server'),
```

Comment:
These changes are all needed as with this PR we're freeing memory is different subsystems to where we're allocating it.  We should probably just remove this check entirely at this point.